script/chests.js

```diff
@@ -598,17 +611,17 @@ var dungeons = [
             'Child Shooting Gallery': { isAvailable: function () {
                 return true; }, },
             'Bombchu Bowling 1': { isAvailable: function () {
-                return items.Bombs; }, },
+                return items.Bombchus; }, },
```

Comment:
This is not correct. If Bombchus in logic is off, then you need the bomb bag, not chus

Reply:
I have no idea why I did that. LOL Will Fix...